regression/cbmc-shadow-memory/var-assign2/test.desc

```diff
@@ -0,0 +1,8 @@
+KNOWNBUG
+main.c
+
+^EXIT=0$
+^SIGNAL=0$
+^VERIFICATION SUCCESSFUL$
+--
+^warning: ignoring
```

Comment:
Can we add an explanation here or in the source code about why this is a bug/problem? It would be great not to lose the knowledge (or only keeping it private).

Reply:
Sorry that test belong to a commit that has been dropped as its behaviour has not been decided yet.

It has been moved to a different branch that will be addressed separately.